src/components.ts

```diff
@@ -90,7 +92,11 @@ export async function initComponents(): Promise<AppComponents> {
     const dbPassword = await config.requireString('PG_COMPONENT_PSQL_PASSWORD')
     databaseUrl = `postgres://${dbUser}:${dbPassword}@${dbHost}:${dbPort}/${dbDatabaseName}`
   }
+  const serverName = 'social-service'
   const privateVoiceChatJobInterval = await config.requireNumber('PRIVATE_VOICE_CHAT_JOB_INTERVAL')
+  const analyticsApiUrl = await config.requireString('ANALYTICS_API_URL')
+  const analyticsApiToken = await config.requireString('ANALYTICS_API_TOKEN')
+  const env = (await config.requireString('ENV')) as Environment
```

Comment:
WDYT about getting the values of all these env vars related to the analytics component (not the job interval) inside the component?

Reply:
Done!